src/main/java/uk/gov/hmcts/ccd/domain/service/caseaccess/CaseAccessOperation.java

```diff
@@ -479,12 +479,14 @@ public void removeCaseUserRoles(List<CaseAssignedUserRoleWithOrganisation> caseU
     }
 
     private void validateCaseRoles(Set<String> validCaseRoles, Set<String> targetCaseRoles) {
-        targetCaseRoles.stream()
-            .filter(role -> !validCaseRoles.contains(role))
-            .findFirst()
-            .ifPresent(role -> {
-                throw new InvalidCaseRoleException(role);
-            });
+        if (targetCaseRoles != null) {
```

Comment:
If **targetCaseRoles** comes to this part as null, it should be failed, even with NPE. Otherwise, this would continue working, but this is not the purpose of the code.